.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,20 @@
+*Please explain what this PR does here.*
+
+#### Checklist
+
+Before merging this PR, the following things must have been done:
+
+- [ ] Faithful integration of the mockups at all screen sizes
+- [ ] Tested on supported browsers, including responsive mode (IE11 and Edge, Chrome >= 42, 2 latest versions of Firefox and Safari)
```

Comment:
I think the list of supported browsers should be updated (**IE11**, cough)

Reply:
https://github.com/cozy/create-cozy-app/issues/1293